app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -551,12 +584,14 @@ private fun PlantDescription(description: String) {
 @Preview
 @Composable
 private fun PlantDetailContentPreview() {
+    // FIXME: Preview is broken because of GlideImage. See: https://github.com/bumptech/glide/issues/4977
```

Comment:
one thing we could do is wrap GlideImage in our own composable and use `LocalInspectionMode` to render e.g. red Box .. this way, the previews are working until the issue is resolved

Reply:
Good idea! Will create a wrapper for `GlideImage`